code/api/views.py

```diff
@@ -13,10 +16,12 @@
 import logging
 
 from cmput404.constants import HOST, API_PREFIX
+from socialDistribution.forms import PostForm
```

Comment:
Do we need this import? Just asking because it's for socialDistribution but we're in the api app.